ios/RNGalleryManager.m

```diff
@@ -130,19 +138,10 @@ @implementation RNGalleryManager
   NSArray<PHAsset*>  *results = [fetchResults objectsAtIndexes:indexSet];
   
   for (PHAsset* asset in results) {
-    NSArray *resources = [PHAssetResource assetResourcesForAsset:asset ];
-    if ([resources count] < 1) continue;
-    NSString *orgFilename = ((PHAssetResource*)resources[0]).originalFilename;
-    NSString *uit = ((PHAssetResource*)resources[0]).uniformTypeIdentifier;
-    NSString *mimeType = (NSString *)CFBridgingRelease(UTTypeCopyPreferredTagWithClass((__bridge CFStringRef _Nonnull)(uit), kUTTagClassMIMEType));
-    CFStringRef extension = UTTypeCopyPreferredTagWithClass((__bridge CFStringRef _Nonnull)(uit), kUTTagClassFilenameExtension);
-    
     [assets addObject:@{
                         @"type": [self getMediaType:([asset mediaType])],
                         @"width": @([asset pixelWidth]),
                         @"height": @([asset pixelHeight]),
-                        @"filename": orgFilename ?: @"",
-                        @"mimeType": mimeType ?: @"",
```

Comment:
My application depends on the mime type...

Reply:
ok you can keep mimeType then.